src/librustc_resolve/lib.rs

```diff
@@ -257,7 +257,7 @@ fn resolve_struct_error<'b, 'a: 'b, 'c>(resolver: &'b Resolver<'a>,
                              "type `{}` is not a member of trait `{}`",
                              type_,
                              trait_);
-            err.span_label(span, &format!("not a member of trait `Foo`"));
+            err.span_label(span, &format!("not a member of `{}`", trait_));
```

Comment:
Heh, you and @GuillaumeGomez made the same comment at the same time.

@nagisa and I talked about it on IRC -- the same error for methods already doesn't say "trait" in the label. Either way they should be consistent. I'm fine with putting it back in -- I thought we should err on the side of shorter span labels. Note that the main error message _does_ include the word "trait".